src/main/java/com/powsybl/ws/commons/computation/service/AbstractWorkerService.java

```diff
@@ -141,7 +150,12 @@ public Consumer<Message<String>> consumeRun() {
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt();
             } catch (Exception e) {
-                if (!(e instanceof CancellationException)) {
+                if (isFailureWithLogs(e)) {
+                    LOGGER.error(NotificationService.getFailedMessage(getComputationType()), e);
```

Comment:
Maybe we can get common code out of the if() (above and below) to see more clearly the logic?